README.md

```diff
@@ -9,35 +9,42 @@ Prerequisites
 - [Go](https://golang.org/).
 - The [GOPATH environment variable](https://github.com/golang/go/wiki/GOPATH) correctly set.
 
+An example setup:
+
+```
+export GOROOT="$HOME/opt/go1.4.2"
```

Comment:
You're not supposed to set GOROOT directly unless you are doing something outside the defaults with your go installation. See (and possibly link to) https://github.com/golang/go/wiki/InstallTroubleshooting